managed/services/agents/node.go

```diff
@@ -44,8 +44,9 @@ func nodeExporterConfig(node *models.Node, exporter *models.Agent, agentVersion
 
 		"--web.disable-exporter-metrics", // we enable them as a part of HR metrics
 
-		"--web.listen-address=:" + tdp.Left + " .listen_port " + tdp.Right,
-	}
+//		"--web.listen-address=:" + tdp.Left + " .listen_port " + tdp.Right,
+                "--web.listen-address=" + listenInterface + ":" + tdp.Left + " .listen_port " + tdp.Right,
```

Comment:
It should be declared here as placeholder, same as `.listen_port`. And then you pmm-agent should fill it with proper interface. Otherwise it will be the same interface for all nodes.